<launch>

  <arg name="carrot_dist" default="10."/>
  <arg name="base_frame" default="sam/base_link"/>
  <arg name="thrust_level" default="200."/>
  <arg name="heading_offsets" default="5."/>
  <arg name="trajectory_points" default="100"/>
  <arg name="goal_tolerance" default="3."/>

  <!-- <include file="$(find auv_simple_motion_planner)/launch/controller.launch">
    <arg name="carrot_dist" value="$(arg carrot_dist)" />
    <arg name="base_frame" value="$(arg base_frame)" />
    <arg name="thrust_level" value="$(arg thrust_level)" />
  </include>
 -->
<!--   <node name="bezier_planner" pkg="auv_simple_motion_planner" type="bezier_planner.py" output="screen">
    <param name="goal_tolerance" value="$(arg goal_tolerance)" />
    <param name="heading_offsets" value="$(arg heading_offsets)" />
    <param name="base_frame" value="$(arg base_frame)" />
    <param name="number_points" value="$(arg trajectory_points)" />
  </node> -->

  <node name="p2p_planner" pkg="auv_simple_motion_planner" type="p2p_planner.py" output="screen">
    <param name="goal_tolerance" value="$(arg goal_tolerance)" />
    <param name="heading_offsets" value="$(arg heading_offsets)" />
    <param name="base_frame" value="$(arg base_frame)" />
  </node>

<node name="manual_bt" pkg="waypoints" type="manual_bt.py" output="screen"/>   

</launch>
